docs/INSTALL.md

```diff
@@ -172,66 +172,71 @@ In this case the filesystem will be mounted at startup, and the sfs process will
 Sync daemon component
 ===============
 
-Configuring php-sync on the central node
+Prerequisites
+--------------------
+Install php, if not already present.
+
+Make sure you have installed php together with:
```

Comment:
does debian have only one big php install file?
On php.net these modules are marked as optional, so they may not be installed. I can name this after the php modules that have to be present.